x-pack/test/api_integration/apis/monitoring/apm/index.js

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+export default function ({ loadTestFile }) {
+  describe('Apm', () => {
```

Comment:
Not as critical as the other ones, but probably best to be consistent. Apm -> APM